botfront/imports/ui/components/utils/IntentLabel.jsx

```diff
@@ -44,7 +46,13 @@ function Intent({
         >
             <Popup
                 trigger={(
-                    <Label id='intent' color='purple' data-cy='intent-label' size={size}>
+                    <Label
+                        id='intent'
+                        color={value !== OOS_LABEL ? 'purple' : 'grey'}
+                        basic={value === OOS_LABEL}
+                        data-cy='intent-label'
+                        size={size}
+                    >
```

Comment:
Remove the out of scope logic from here

Reply:
We need it to show (pick intent) with different styling

Reply:
mmh ok